APE14.rst

```diff
@@ -195,14 +232,18 @@ low-level API recommends:
             Convert pixel coordinates to world coordinates. This method takes
             n_pixel scalars or arrays as input, and pixel coordinates should be
             zero-based. Returns n_world scalars or arrays in units given by
-            ``world_axis_units``.
+            ``world_axis_units``. Note that pixel coordinates are assumed
+            to be such that they are 0 at the center of the first pixel in each
```

Comment:
` to be such that they are 0` -> ` to be 0`

Reply:
Fixed